### PR TITLE
Slurm6. Add basic example and integration test

### PR DESCRIPTION
**Pull request by mr0re1**
* Add Slurm6 example to community;
* Add integration test for this example.

**NOTE**: `network_storage` on partition nodes are flaky, do not include it into example/test. Add after fixed.

### CODE REVIEW

File: tools/cloud-build/daily-tests/tests/slurm-v6-rocky8.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+# non-alphanumeric chars and is capped at 10 chars.
+slurm_cluster_name: "rock8{{ build[0:5] }}"
+
+cli_deployment_vars:
+   region: us-west4
+   zone: us-west4-c
+
+zone: us-west4-c
+workspace: /workspace
+blueprint_yaml: "{{ workspace }}/community/examples/hpc-slurm-ubuntu2004.yaml"
```
Comment (harshthakkar01) [inline tools/cloud-build/daily-tests/tests/slurm-v6-rocky8.yml:29]: this should be hpc-slurm6.yaml

File: community/examples/hpc-slurm6.yaml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+vars:
+  project_id:  ## Set GCP Project ID Here ##
+  deployment_name: slurm-gcp-v6
+  region: us-west4
+  zone: us-west4-c
+  instance_image:
+    family: slurm-gcp-6-1-hpc-rocky-linux-8
+    project: schedmd-slurm-public
+
```
Comment (harshthakkar01) [inline community/examples/hpc-slurm6.yaml:27]: nit: extra new line.

**Comment by harshthakkar01** [comment]
Also, do we want to put this test in babysit ?

File: community/examples/hpc-slurm6.yaml (hunk 2 above)
Comment (mr0re1) [reply community/examples/hpc-slurm6.yaml:27]: Thank you for spotting this )

**Comment by mr0re1** [comment]
> Also, do we want to put this test in babysit ?

No, we don't

File: tools/cloud-build/daily-tests/tests/slurm-v6-rocky8.yml (hunk 1 above)
Comment (mr0re1) [reply tools/cloud-build/daily-tests/tests/slurm-v6-rocky8.yml:29]: Thank you for spotting this )

**Review by harshthakkar01 — APPROVED** [review]

**Pull request merged**